class/Zarieth; The Witch

```diff
@@ -0,0 +1,1836 @@
+{
+	"_meta": 
+	{
+		"sources": 
+		[
+			{
+				"json": "ZariethTheWitchFromGMBinder",
+				"abbreviation": "Brew",
+				"full": "The Witch",
+				"authors": 
+				[
+					"u/Zarieth"
+				],
+				"version": "1.1",
+				"url": "https://www.gmbinder.com/share/-L7fK9yKzbw1PUOM3iJi",
+				"targetSchema": "1.0.0"
+			}
+		],
+		"dateAdded": 1549127108
+	},
+	"class": 
+	[
+		{
+			"name": "The Witch",
+			"source": "The Witch",
```

Comment:
This should match your `"json"` source defined in `"_meta"`, e.g. be `"ZariethTheWitchFromGMBinder"` (and the same for the subclass sources)

Reply:
Changed in all instances. Thanks for the Feedback.